OpenRA.Mods.Common/Traits/BotModules/BaseBuilderBotModule.cs

```diff
@@ -277,6 +277,9 @@ int MinimumRefineryCount
 		{
 			get
 			{
+				if (Info.RefineryTypes.Count() == 0)
```

Comment:
`MinimumRefineryCount` only seems to be used by `HasAdequateRefineryCount`, which already checks if we have any refinery types, so is this check still necessary?

Reply:
You are right this is covered by https://github.com/OpenRA/OpenRA/pull/18484 already although I think this is the better place for it.